### PR TITLE
Add double click logic in scg view

### PR DESCRIPTION
**Pull request by smousy**

### CODE REVIEW

**Review by KovalM — CHANGES_REQUESTED** [review]
1. Try checkout to SCg and checkout to SCn. You will get constantly loading
2. Try checkout to SCg, do double click to any node, checkout to SCn. You will get lang checkout

**Review by KovalM — APPROVED** [review]

File: client/js/Ui/windowmanager.js [hunk 1]
```diff
@@ -115,9 +116,15 @@ SCWeb.ui.WindowManager = {
         this.history_tabs.prepend(tab_html);
 
         // get translation and create window
-        var ext_lang_addr = SCWeb.core.Main.getDefaultExternalLang();
-        command_state.format = SCWeb.core.ComponentManager.getPrimaryFormatForExtLang(ext_lang_addr);
-        command_state.lang = SCWeb.core.Translation.getCurrentLanguage();
+        if(!command_state.format)
```
Comment (MikhailSadovsky) [inline client/js/Ui/windowmanager.js:119]:
```suggestion
        if (!command_state.format)
```

File: client/js/Ui/windowmanager.js [hunk 2]
```diff
@@ -115,9 +116,15 @@ SCWeb.ui.WindowManager = {
...
+        {
+            var ext_lang_addr = SCWeb.core.Main.getDefaultExternalLang();
+            command_state.format = SCWeb.core.ComponentManager.getPrimaryFormatForExtLang(ext_lang_addr);
+        }
+
+        if(!command_state.lang)
```
Comment (MikhailSadovsky) [inline client/js/Ui/windowmanager.js:125]:
```suggestion
        if (!command_state.lang)
```

**Review by MikhailSadovsky — CHANGES_REQUESTED** [review]

**Review by MikhailSadovsky — APPROVED** [review]

File: client/js/Core/main.js [hunk 3]
```diff
@@ -208,7 +208,6 @@ SCWeb.core.Main = {
      * @param {Array} cmd_args Array of sc-addrs with command arguments
```
Comment (NikitaZotov) [inline client/js/Core/main.js:208]: Add info about this feature into changelog, please

**Review by NikitaZotov — APPROVED** [review]

**Pull request merged**